cdap-ui/app/cdap/components/OpsDashboard/RunsGraph/graphRenderer.js

```diff
@@ -136,20 +143,40 @@ export function renderGraph(selector, containerWidth, containerHeight, data, vie
   dateAxisGroup.select('.domain')
     .remove();
 
+  const legendOffset = 30;
+  const ticksFontSize = 10;
+
   // Y Axis Left
   chart.append('g')
     .attr('class', 'axis axis-y-left')
     .call(d3.axisLeft(yLeft).tickFormat((e) => {
       // showing only integers
       if (Math.floor(e) !== e) { return; }
       return e;
-    }));
+    }))
+    .attr('font-size', ticksFontSize);
```

Comment:
there's already a class in here, is it possible to set the font-size using the class?

Reply:
So I added this here so that the legend calculation on line 178 makes sense, instead of just pulling a number out of nowhere. The default tick font size is already 10px. But in the future, if we want to change the tick font size, then we just have to change this variable, and the legend will still be placed correctly.

Reply:
move these `.attr` before the `.call` for better organization